frontend/cypress/utility/randomizer.js

```diff
@@ -0,0 +1,18 @@
+class Randomizer {
+  createAlphanumeric(desiredLength) {
+    let text = "";
+    const possible =
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+    for (let i = 0; i < desiredLength; i++)
+      text += possible.charAt(Math.floor(Math.random() * possible.length));
+    return text;
+  }
+  createNumeric(desiredLength) {
+    let text = "";
+    const possible = "0123456789";
+    for (let i = 0; i < desiredLength; i++)
+      text += possible.charAt(Math.floor(Math.random() * possible.length));
```

Comment:
nebūtų šiek tiek našiau šitą daryt su `Math.random() * 9 * Math.pow(10, desiredLength) +  Math.pow(10, desiredLength - 1) - 1`
arba kažkas tokio 😆